flutter/test/mocks.mocks.dart

```diff
@@ -1,11 +1,10 @@
-// Mocks generated by Mockito 5.3.1 from annotations
+// Mocks generated by Mockito 5.2.0 from annotations
```

Comment:
If this is needed, upgrade your local mockito version and run it again, otherwise, just revert these changes.